src/documents/firstDiscoveryServer.md

```diff
@@ -0,0 +1,365 @@
+---
+title: First Discovery Server
+layout: default
+category: Overview
+---
+
+## Launching ##
+
+The First Discovery Server can be launched as a Kettle application by making use of [Kettle Configs](https://github.com/amb26/kettle/blob/KETTLE-32/README.md#structure-of-a-kettle-config).
```

Comment:
This is the very first mention ever of Kettle. Perhaps the first occurrance of the word should be a link to some root documentation? perhaps https://github.com/amb26/kettle/blob/KETTLE-32/README.md ?